Libraries/LibWeb/WebIDL/AbstractOperations.cpp

```diff
@@ -517,6 +517,14 @@ JS::ThrowCompletionOr<T> convert_to_int(JS::VM& vm, JS::Value value, EnforceRang
 
         // 2. Round x to the nearest integer, choosing the even integer if it lies halfway between two, and choosing +0 rather than −0.
         // 3. Return x.
+        // NOTE: The C function lround matches the web standard except for the middle halfway case.
+        //       C rounds away from 0. We need to correct for that particular case.
+        if (ceil(x) - x == 0.5L) {
+            if (lround(x) % 2 == 0)
+                return round(x);
+            return round(x) - 1;
+        }
```

Comment:
Isn't rounding to even integers in the halfway case a typical supported rounding mode? E.g. `std::fesetround` with `FE_TONEAREST`.

We tend not to use the standard namespace though, however we might want to introduce something for this so we can use our AK's `round_to<T>` together with a specified rounding mode?